spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncher.java

```diff
@@ -182,20 +236,50 @@ private void launchModule(String module, String[] args) {
 		}
 	}
 
-	private Resource resolveModule(String coordinates) {
+	private void launchModuleWithDependencies(String module, String[] args, String[] includes, String[] excludes) {
+		try {
+			Resource[] libraries = this.moduleResolver.resolve(
+					new Coordinates[]{toCoordinates(module, DEFAULT_MODULE_CLASSIFIER)},
```

Comment:
I know. Currently we're ignoring the contents of `/lib/*.jar`, which is suboptimal. At least for a first step, I'd prefer to keep things as close as possible to the original and have this as a future enhancement.